qtpy/uic.py

```diff
@@ -180,8 +181,12 @@ def _get_custom_widgets(ui_file):
             return {}
 
         custom_widget_classes = {}
+        if PY33:
```

Comment:
We are not supporting this version of python, so why do we need this?

Reply:
Sorry for the late reply. PY33 is a variable to ensure the python version is 3.3 and above. As per the below deprecation it seems the methods were deprecated from 3.2. So I made sure to use `list(custom_widgets)` in Python 3.3+ and fallback to `getchildren` for versions below python 3.3 . Python 3.3 is listed in setup.py and this is the format used in most places for compatibility and hence my change.

> Methods getchildren() and getiterator() of classes ElementTree and Element in the ElementTree module have been removed. They were deprecated in Python 3.2. Use iter(x) or list(x) instead of x.getchildren() and x.iter() or list(x.iter()) instead of x.getiterator(). (Contributed by Serhiy Storchaka in bpo-36543.)

https://docs.python.org/3.9/whatsnew/3.9.html#removed